index.html

```diff
@@ -77,7 +77,7 @@
         <div class="col-md-4">
           <h2>Heading</h2>
           <p>Donec sed odio dui. Cras justo odio, dapibus ac facilisis in, egestas eget quam. Vestibulum id ligula porta felis euismod semper. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus.</p>
-          <p><a class="btn btn-default" href="#" role="button">View details &raquo;</a></p>
+          <p><a class="btn btn-default" href="http://www.publicdomainpictures.net/browse-category.php?c=people&s=8" role="button">View details &raquo;</a></p>
```

Comment:
Public domain link looks good!  I agree that we should link categories to poetry author section.